packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -732,8 +734,8 @@ export class DeltaManager
 
         if (this.readonly === true) {
             assert(this.readOnlyInfo.readonly === true, 0x1f0 /* "Unexpected mismatch in readonly" */);
-            const error = new LoggingError("Op is sent in read-only document state", {
-                errorType: ContainerErrorType.genericError,
+            const error = new GenericError("Op is sent in read-only document state", {
+                errorType: ContainerErrorType.genericError,  //* use GenericError ctor?
```

Comment:
May I suggest using PR comments instead of putting them in code? If the intention is not to merge them.